src/test/java/org/opentripplanner/middleware/tripmonitor/jobs/CheckMonitoredTripBasicTest.java

```diff
@@ -181,4 +182,107 @@ private static Stream<Arguments> createShouldAdvanceToNextDayCases() {
             Arguments.of(-ONE_HOURS_IN_SECS, true, "Should advance monitored day if trip is past.")
         );
     }
+
+    @ParameterizedTest
+    @MethodSource("shouldSendInitialReminderCases")
+    void testShouldSendInitialReminder(
+        int offsetSeconds,
+        boolean active,
+        boolean snoozed,
+        TripStatus tripStatus,
+        boolean expected,
+        String message
+    ) throws CloneNotSupportedException {
+        MonitoredTrip trip = makeMonitoredTripFromNow(offsetSeconds, offsetSeconds + 300);
+        if (tripStatus != TripStatus.PAST_TRIP) {
+            setRecurringTodayAndTomorrow(trip);
+        } else {
+            trip.updateAllDaysOfWeek(false);
+        }
+        trip.isActive = active;
+        trip.snoozed = snoozed;
+        trip.notifyAtLeadingInterval = true;
+        trip.leadTimeInMinutes = 30;
+        trip.journeyState.lastCheckedEpochMillis = Instant.now().minus(60, ChronoUnit.MINUTES).toEpochMilli();
+        trip.journeyState.tripStatus = tripStatus;
+
+        CheckMonitoredTrip check = new CheckMonitoredTrip(trip);
+        check.matchingItinerary = trip.itinerary;
+        check.previousMatchingItinerary = trip.itinerary;
+
+        assertEquals(
+            expected,
+            check.shouldSendInitialReminder(),
+            message
+        );
+    }
+
+    static Stream<Arguments> shouldSendInitialReminderCases() {
+        return Stream.of(
+            Arguments.of(
+                300,
+                true,
+                false,
+                TripStatus.TRIP_UPCOMING,
+                true,
+                "Send reminder today for upcoming monitored trip today."
+            ),
+            Arguments.of(
+                -60,
+                true,
+                false,
+                TripStatus.TRIP_ACTIVE,
+                true,
+                "Send reminder for ongoing monitored trip (for trips is saved after starting time)."
```

Comment:
Nit. `(for trips saved after starting time)`

Reply:
Updated in https://github.com/ibi-group/otp-middleware/pull/332/commits/9381a2f0c299965e60ecddbbd42f17883e048bdf